Enterprise-CMCS/macpro-mdct-mfp
# Cmdct 3115 - fix ability to add and delete initiatives

### Description
After submitting a workplan and having an admin unlock it, a state user couldn't delete initiatives or create or delete new nested initiatives. The issue was due to the fact that every update and delete was sending a full payload with the following 'blacklist` data, which isn't allowed after a workplan has already been submitted/is in revision:
```
    "submitterName",
    "submitterEmailAddress",
    "reportSubmissionDate",
```

### Related ticket(s)


CMDCT-3115

---
### How to test
Create a wp and submit it
login as an admin and unlock it
login as a state user and enter the unlocked workplan
verify that you can delete initiatives and create new ones at the top level and nested level (evaluation plans, funding sources)


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
